app/utility/transforms/CollapsedMetrics.js

```diff
@@ -3,6 +3,7 @@ import {
     EMPTY_METRIC,
     AVG_VELOCITY_METRIC,
     RPE_METRIC,
+    RPE1RM_METRIC,
```

Comment:
- [ ] This also needs UNITS for the 1RM estimation in kg/lb, which is what the current weight is in